modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -7475,6 +7475,67 @@ public void testPostCustomObjectEntryWithNestedCustomObjectEntriesInOneToManyRel
 		_assertEquals(nestedObjectEntriesJSONArray);
 	}
 
+	@Test
+	public void testPostCustomObjectEntryWithNestedPermissions()
```

Comment:
The concept of `nested` only exists in the `GET` method. In `POST/PUT/PATCH` is a property of the Object Entry.

Can you rename this method to testPostCustomObjectEntryWithPermissions please?

Reply:
rename the private method please.